test/xunit.analyzers.tests/xunit.runner.json

```diff
@@ -1,3 +1,3 @@
 ﻿{
-  "appDomain": "denied"
```

Comment:
Can you clarify the specific form you expect to see in this file?

Reply:
When I add this property back, the net472 tests stop working. 😕

Reply:
What OS are you testing on? I'm not seeing failures when disabling app domains on Windows or Ubuntu.

Reply:
This is in Visual Studio 2019 on Windows

Reply:
I see this error running the `net472` tests in VS2017:

```
System.MissingMethodException : Method not found: 'Microsoft.CodeAnalysis.Host.Mef.MefHostServices Microsoft.CodeAnalysis.Host.Mef.MefHostServices.Create(System.Composition.CompositionContext)'.
```

This sounds like a Visual Studio bug and should be reported to them (especially since the tests run just fine with `dotnet test`, as that in theory uses the same VSTest engine as VS2017/VS2019).

Reply:
Either that, or your testing library doesn't work without app domains, which is going to be a problem as we're removing app domains entirely from v3.

Reply:
The changes to this file have now been reverted. I'll take up the test runner issue with the testing team separately.